decidim-meetings/app/commands/decidim/meetings/admin/create_meeting.rb

```diff
@@ -5,83 +5,49 @@ module Meetings
     module Admin
       # This command is executed when the user creates a Meeting from the admin
       # panel.
-      class CreateMeeting < Decidim::Command
-        def initialize(form)
-          @form = form
-        end
-
-        # Creates the meeting if valid.
-        #
-        # Broadcasts :ok if successful, :invalid otherwise.
-        def call
-          return broadcast(:invalid) if @form.invalid?
-
-          transaction do
-            create_meeting!
-            create_services!
-          end
+      class CreateMeeting < Decidim::Commands::CreateResource
+        protected
 
+        def run_after_hooks
+          create_services!
           create_follow_form_resource(form.current_user)
-          broadcast(:ok, meeting)
         end
 
-        private
-
-        attr_reader :form, :meeting
-
-        def create_meeting!
+        fetch_form_attributes :scope, :category, :end_time, :start_time, :online_meeting_url, :registration_type,
```

Comment:
Can we move the `fetch_form_attributes` before the `run_after_hooks` method? I think it makes more sense to first see the attributes and then what will happen afterwards

Reply:
fixed by [e3ac071](https://github.com/decidim/decidim/pull/12191/commits/e3ac071381996529ed484b14cd668b192e8be8bd)